Commands/MessageCount/MessageCountSystem.go

```diff
@@ -5,13 +5,37 @@
 		-msgcount
 */}}
 
-{{$x:=dbIncr .User.ID "msgCount" 1}}{{$list:=""}}
-{{if reFind `\A-m(?:essages?|sg)?c(?:ount)?` .Message.Content}}
-	 You have typed {{$x}} messages.
-{{end}}
-{{if reFind `\A-t(?:op)?m(?:essages?|sg)?` .Message.Content}}
-	{{range dbTopEntries "msgCount" 10 0}}
-		{{- $list =print $list .User.Username " - " .Value "\n" -}}
-	{{end}}
-	{{sendMessage nil (cembed "title" "Message Count Leaderboard" "description" $list)}}
-{{end}}
+{{/* DONT TOUCH UNLESS YOU KNOW WHAT UR DOING c: */}}
+{{/* define variables */}}
+{{ $avatar := (joinStr "" "https://cdn.discordapp.com/avatars/" (toString .User.ID) "/" .User.Avatar ".png") }}
+{{ $x: = dbIncr .User.ID "msgCount" 1 }}{{ $list:="" }}
+{{ $embed := cembed "thumbnail" (sdict "url" $avatar) "title" "Message Counter" "description" (print .User.Mention " have sent " $x  " message(s) ") "color" 0xFF0000 }}
+{{ if reFind `\A\-m(?:essages?|sg)?c(?:ount)?` .Message.Content }} {{/* msgcount command */}}
```

Comment:
ye the regex is actually somewhat inaccurate in comparison to yaggy's usual commands, altho it is my fault for making it lazily, one thing I would add is (?i) at the start for case insensitivity tho, and if you really want to go crazy with the regex: (?i)\A(?:\-|<@!?204255221017214977>)\s*m(?:sgs?|essages?)?-?c(?:ount)?(?:\s|\z)